illumos-utils/src/smf_helper.rs

```diff
@@ -77,7 +77,7 @@ impl<'t> SmfHelper<'t> {
                 "addpropvalue",
                 &prop.to_string(),
                 &format!("{}:", valtype.to_string()),
-                &val.to_string(),
+                &format!("\"{}\"", val.to_string()),
```

Comment:
This is needed to allow the settings of property values containing spaces, e.g., "oxide switch 1".